packages/components/src/components/form/BottomText.tsx

```diff
@@ -30,30 +30,63 @@ export const Container = styled.div<{ $inputState?: InputState; $isDisabled?: bo
         $isDisabled ? theme.textDisabled : getInputStateTextColor($inputState, theme)};
     ${typography.label}
     animation: ${slideDown} 0.18s ease-in-out forwards;
+    background-color: ${({ $inputState, theme }) =>
+        $inputState === 'info' && theme.backgroundAlertBlueSubtleOnElevation1};
+    border-radius: ${borders.radii.xxs};
+
+    ${({ $inputState, theme }) =>
+        $inputState === 'info' &&
+        `
+        margin: ${spacingsPx.sm} 0;
+        padding: ${spacingsPx.xs} ${spacingsPx.sm};
+        ::before {
+            content: '';
+            position: absolute;
+            top: -${spacingsPx.xs};
+            left: 20px;
+            transform: translateX(-50%);
+            width: 0;
+            height: 0;
+            border-left: 8px solid transparent; 
+            border-right: 8px solid transparent; 
+            border-bottom: 8px solid ${theme.backgroundAlertBlueSubtleOnElevation1}; 
+        }
+        `}
```

Comment:
I believe this shouldn't be here, because `BottomText` is generic component in the design system. 

I suggest – if you want to use some custom component – just pass `bottomText`: (as custom ReactNode not string) to `Input` and style it on your own outside of the design system.

Something like:

```tsx
<Input 
  ...
  bottomText={<CustomComponent/>}
/>
```

Reply:
This is removed and a custom component is introduced